recipes/libensemble/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "libensemble" %}
+{% set version = "0.6.0" %}
+{% set build = 0 %}
+{% set sha256 = "bf0483a05bfac383f82fd2a79ca5151af7a2337e07b2392eee2e6396c250357e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/Libensemble/libensemble/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - nlopt
+    - numpy
+    - petsc
+    - petsc4py
+    - python
+    - scipy
+
+test:
+  imports:
+    - opmd_viewer
+
+about:
+  home: https://www.openPMD.org
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "A library to coordinate the concurrent evaluation of dynamic ensembles of calculations"
+
+  description: |
+    libEnsemble is a Python library to coordinate the concurrent evaluation
+    of dynamic ensembles of calculations. The library is developed to use
+    massively parallel resources to accelerate the solution of design,
+    decision, and inference problems and to expand the class of problems that
+    can benefit from increased concurrency levels.
+    libEnsemble aims for: Extreme scaling, Resilience/fault tolerance,
+    Monitoring/killing jobs (and recovering resources), Portability and
+    flexibility, Exploitation of persistent data/control flow.
+  dev_url: https://github.com/Libensemble/libensemble
+
+extra:
+  recipe-maintainers:
+    - ax3l
+    - shuds13
```

Comment:
cc @shuds13: I'll give more details in our meeting later today. who else want's to co-maintain this automated feedstock (by github handle)?

Please comment with "please add me" here.

Reply:
Please add jlnav

Reply:
I am happy to be listed as a package maintainer.

Reply:
Please add me

Reply:
@ax3l, was it intentional that your name isn't here?

Reply:
Yep, I am a downstream consumer :)

Reply:
@isuruf ready for merge :)